pilot/pkg/model/gateway.go

```diff
@@ -87,10 +90,21 @@ func RecordRejectedConfig(gatewayName string) {
 	totalRejectedConfigs.With(typeTag.Value("gateway"), nameTag.Value(gatewayName)).Increment()
 }
 
+// DisableGatewayPortTranslationLabel is a label on Service that declares that, for that particular
+// service, we should not translate Gateway ports to target ports. For example, if I have a Service
+// on port 80 with target port 8080, with the label. Gateways on port 80 would *not* match. Instead,
+// only Gateways on port 8080 would be used. This prevents ambiguities when there are multiple
+// Services on port 80 referring to different target ports. Long term, this will be replaced by
+// Gateways directly referencing a Service, rather than label selectors. Warning: this label is
```

Comment:
Is this k8s Gateway or istio Gateway

Reply:
The label - for Istio gateway. Note it's not new in this PR, just moved

The "long term" part of the comment - it will be k8s gateway, but we may backport it to Istio gateway possibly